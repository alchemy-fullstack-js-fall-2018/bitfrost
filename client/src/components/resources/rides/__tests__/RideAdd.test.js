import React from 'react';
import { shallow } from 'enzyme';
import { RideCreate } from '../RideCreate.jsx';

describe('<RideCreate />', () => {
  test('renders', () => {
    const fetchCar = jest.fn();
    const user = { _id: 'abc' };

    const wrapper = shallow(
      <RideCreate
        fetchCar={fetchCar}
        user={user}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
