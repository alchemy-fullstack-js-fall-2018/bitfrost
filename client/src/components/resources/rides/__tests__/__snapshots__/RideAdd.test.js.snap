// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RideCreate /> renders 1`] = `
<styled.div>
  <Nav
    pageTitle="Create A Trip"
  />
  <styled.div>
    <SearchMap
      handlePositions={[Function]}
    />
  </styled.div>
  <styled.form
    onSubmit={[Function]}
  >
    <styled.div>
      <label
        htmlFor="departDTL"
      >
        Estimated Depart Time: 
      </label>
      <input
        name="departDTL"
        onChange={[Function]}
        type="datetime-local"
        value=""
      />
    </styled.div>
    <styled.div>
      <label
        htmlFor="arriveDTL"
      >
        Estimated Arrival Time: 
      </label>
      <input
        name="arriveDTL"
        onChange={[Function]}
        type="datetime-local"
        value=""
      />
    </styled.div>
    <styled.div>
      <styled.button
        type="submit"
      >
        Create new trip
      </styled.button>
    </styled.div>
  </styled.form>
</styled.div>
`;
